server/src/db/dal/init-db.ts

```diff
@@ -3,46 +3,57 @@ import { Category } from "../schemas/category.schema";
 
 const categories: Category[] = [{
     name: 'לבוש',
+    sentenceBeginning: 'אני רוצה ללבוש',
```

Comment:
You have two `sentenceBeginning` fields :O
"מחכה ששוקר תסיים לעשות התחלות משפטים"
חחחח